examples/cookie-sharing/background.js

```diff
@@ -0,0 +1,10 @@
+chrome.action.onClicked.addListener(async (tab) => {
+  const url = "https://crx-cookie-sharing.glitch.me/dreams";
+  const res = await fetch(url, {credentials: "include"});
+
+  let cookies = await chrome.cookies.getAll({
+    domain: chrome.runtime.getURL(''),
+  });
+
+  console.log(cookies);
```

Comment:
I was like — this demo clearly has to do something! No wait, it's just logging nothing to console.

<img width="803" alt="Screen Shot 2021-04-08 at 12 09 34" src="https://user-images.githubusercontent.com/119184/113957811-4d3d9d80-9863-11eb-89ee-2782c34de52a.png">

If I visit the site, _then_ click on the button, I get this error. I'm not a cookie expert. I don't know what to make of this.